storage/database/src/main/java/de/danoeh/antennapod/storage/database/PodDBAdapter.java

```diff
@@ -456,6 +459,7 @@ public void setFeedPreferences(FeedPreferences prefs) {
         values.put(KEY_FEED_SKIP_INTRO, prefs.getFeedSkipIntro());
         values.put(KEY_FEED_SKIP_ENDING, prefs.getFeedSkipEnding());
         values.put(KEY_EPISODE_NOTIFICATION, prefs.getShowEpisodeNotification());
+        values.put(KEY_FEED_NEW_EPISODES_ACTION, prefs.getNewEpisodesAction().ordinal());
```

Comment:
This feels risky (when someone adds new enum fields later). How about adding a second constructor parameter to the enum that contains an integer "code"?

Reply:
Sorry, I don't understand how this would work, and how it would impact all the places where the enum is used in comparisons.

Reply:
My idea would be to do it similarly to the `SortOrder` struct. There each enum item has a field that stores an additional integer "code". The enum then also has a method for converting the integer to the enum. That would make some `if...else` statements here a bit more clean. Also, it would make it easier to convert from the stored value to the enum (related to my other comment about directly setting the global value).

Reply:
Thanks, the last week has been super busy, but I should be able to work on this later this week or on Saturday.

Reply:
Awesome, thanks! By the way, we will have a community meeting today, in case you want to join :) See https://forum.antennapod.org/t/monthly-community-call/1869

Reply:
Sorry, we were out of the house all day on Saturday unexpectedly... working on some of these items now. Should be able to finish sometime early this week, but might need to ask for some help.